src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+| VascularCrushing                  | RECOMMENDED                                                                                                                     | [boolean][]                             | Boolean indicating if Vascular Crushing is used. Corresponds to DICOM Tag 0018,9259 `ASL Crusher Flag`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| AcquisitionVoxelSize              | RECOMMENDED                                                                                                                     | [array][] of [numbers][]                | An array of numbers with a length of 3, in millimeters. This parameter denotes the original acquisition voxel size, excluding any inter-slice gaps and before any interpolation or resampling within reconstruction or image processing. Any point spread function effects, for example due to T2-blurring, that would decrease the effective resolution are not considered here.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I think this could fall under "Common metadata fields" for MRI in general.

Reply:
I might be wrong but I think that @tsalo was also referring to the fact that several items in this table that already exist in the "common metadata" or another table.

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#common-metadata-fields

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#sequence-specifics

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#timing-parameters

For example (I have not checked them all):

- `MagneticFieldStrength` is in the common-metadata-fields table.
- `PulseSequenceType` already exists in sequence-specifics table but the description is different.
- `EchoTime` is in the timing parameters table.
- I suspect `VolumeTiming`, `SliceTiming`, `RepetitionTime` are in the same boat.

I think that if keyname already exists somewhere else AND the description is the same, then it might not be necessary to duplicate them here. If only the requirement level change maybe we can have a lighter version of the table.

I am not sure how I feel about keeping the same keyname if the description for ASL is different than in the field that already exists.

Reply:
> I think that if keyname already exists somewhere else AND the description is the same, then it might not be necessary to duplicate them here. If only the requirement level change maybe we can have a lighter version of the table.

Once we have rendering with the schema fully set up, tables with truly duplicated definitions should be easy enough to build. I think it's just key that we make sure that the definitions are completely the same.

> I am not sure how I feel about keeping the same keyname if the description for ASL is different than in the field that already exists.

I agree. I'm not sure if there's a written rule about this, but my opinion is that descriptions or formats of metadata fields should not vary  depending on the section of the specification, although requirement levels can.

Reply:
Hi @tsalo and @Remi-Gau. This is an issue with which were struglin with for a while, and contacted BIDS maintainers for several times. The definitions are always the same, but there are some additional specifics for ASL. For example, MagneticFieldStrength becomes required (which is needed for the quantifcation of ASL data), PulseSequenceType has to include 2D or 3D, in order to allow conditional check for SliceTiming, etc. I can easily create a document comparing the differences between the definition for 'common' and for 'asl' for each duplicated field, if that helps? Maybe we should go over them one by one, together with some of the maintainers?

Reply:
@patsycle Sorry I just joined the maintainers recently and I did not realize there had been some previous discussions about this so I jumped in a bit too abruptly. :pray: 

I actually do think that have this "difference" document between ASL and common would be useful: it really makes me realize I read this section way too quickly (or that I am just bad at playing "spot the differences" :wink: )

Reply:
@Remi-Gau Haha, no problem. This is, in my opinion, one of the most difficult parts of the extension, and probably a 'main bids topic' which should be cleared out in view of other extensions as well. So I am glad that this question has been raised! I will create this document tomorrow or on Thursday and add the link here. Maybe some kind of 'rule of thumb' can be created this way.

Reply:
Yes @patsycle let's redefine these main parameters in this PR such that it easy to see the difference. I don't think we need a separate document.